cmd/nephe-controller/main.go

```diff
@@ -86,26 +88,46 @@ func main() {
 	// Initialize Account poller map.
 	poller := controllers.InitPollers()
 
-	if err = (&controllers.CloudEntitySelectorReconciler{
+	npController := &controllers.NetworkPolicyReconciler{
 		Client: mgr.GetClient(),
-		Log:    logging.GetLogger("controllers").WithName("CloudEntitySelector"),
+		Log:    logging.GetLogger("controllers").WithName("NetworkPolicy"),
 		Scheme: mgr.GetScheme(),
-		Poller: poller,
-	}).SetupWithManager(mgr); err != nil {
+	}
+
+	if err = npController.SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "NetworkPolicy")
+		os.Exit(1)
+	}
+
+	cesReconciler := controllers.CloudEntitySelectorReconciler{
```

Comment:
keep it consistent and call it controller

Reply:
Done